source/octod/core.d

```diff
@@ -175,12 +175,14 @@ struct HTTPConnection
 
         Params:
             url = GitHub API method URL (relative)
+            accept = optional argument with custom `Accept` header to
+                specify for this one request
 
         Returns:
             Json body of the response. If response is multi-page, all pages
             are collected and concatenated into one returned json object.
      **/
-    Json get ( string url )
+    Json get ( string url, string accept = "" )
```

Comment:
In D there is no real difference between `null` and `""`, (same applies for `null` and empty array). `""` will have an address pointing to `\0` for compatibility with C strings but from the PoV of own D type system both are same arrays with `length == 0`. With that in mind I prefer `""` exactly because it leaves no confusion of `null` being special.